test/rpc-server/get-model/no-predicate/README.md

```diff
@@ -0,0 +1,8 @@
+# `get-model` applied to an input without a predicate
+
+* Input kore term: `\and(\not(X:SortBool), X:SortBool)`
+* Result: `satisfiability-unknown`
+
+The input is interpreted as a _term_, not as a predicate. Therefore,
```

Comment:
Why is input of this test is interpreted as a term (not predicate), and the input of [test/rpc-server/get-model/not-satisfiable-vars](https://github.com/runtimeverification/haskell-backend/pull/3604/files#diff-198d373c2409878031412be16e04863d5e02c8a2dea0dff09410c2759cc224e2) *is* interpreted as a predicate? Is that the `\equals` that makes the second input a predicate?

I'd actually expect a good deal of inputs to this endpoint to be conjunctions of equalities, akin to further tests. Why are these predicates and this one is not?

Reply:
The fact that you can `\and` two terms obfuscates the matter here, but the input is an `\and` of two non-predicates because a variable is not a predicate (predicates being defined as ML constructs whose value is either `\top` or `\bottom`.

Reply:
Thanks! that makes sense now.